.pre-commit-config.yaml

```
@@ -690,6 +690,12 @@ repos:
         files: ^airflow/www/static/js/
         entry: scripts/ci/static_checks/www_lint.sh
         pass_filenames: false
+      - id: changelog-duplicates
+        name: Check changelogs for duplicate entries
+        language: python
+        files: CHANGELOG.txt$|CHANGELOG.rst$
```

Need to escape `.` because this is regex

```suggestion
        files: CHANGELOG\.txt$|CHANGELOG\.rst$
```